core/services/scan.go

```diff
@@ -246,6 +249,11 @@ func (s *ScanService) ScanCVE(ctx context.Context) error {
 		}
 	}
 
+	// if CVE was retrieved from storage, and we don't have a CVE', we send an empty report
+	if cveRetrievedFromStorage && cvep.Content == nil {
```

Comment:
I am not sure I follow.
As I understood the logic should be like
if the "new list of CVEs" ==="known list of CVEs" then don't send

Reply:
we don't store state... so with @dwertent we took this approach instead

Reply:
So what we **do** check in this if statement?

Reply:
what is the meaning of `cvep.Content == nil`?

Reply:
it means we don't have a relevant cve

Reply:
Maybe I need to see it in action to understand the impact.
If we have relevant will we send the same full report over and over?